src/components/Navigation/Navigation.test.jsx

```diff
@@ -1,10 +1,16 @@
 import React from "react";
-import { shallow } from "enzyme";
 import Navigation from "./Navigation";
+import { shallowWithIntl } from "../../utils/intl-enzyme-test-helper";
 
-describe("Navigation", () => {
-  const setup = () => shallow(<Navigation />);
+jest.mock("react-intl", () => ({
+  ...jest.requireActual("react-intl"),
+  useIntl: () => ({
+    formatMessage: jest.fn(),
+  }),
+}));
 
+describe("Navigation", () => {
+  const setup = () => shallowWithIntl(<Navigation />);
```

Comment:
❓ Do we need to add intl stuff here?

Reply:
Addressed:  03bf83ec41b4f47bcfaf65f1e4e940a39c5e251a